lib/oli_web/live/delivery/student/lesson_live.ex

```diff
@@ -1117,6 +1117,7 @@ defmodule OliWeb.Delivery.Student.LessonLive do
       {:noreply,
        socket
        |> assign(page_context: page_context)
+       |> assign(scripts: Utils.get_required_activity_scripts(page_context))
        |> assign(begin_attempt?: true, show_loader?: true)
        |> clear_flash()
        |> assign_html()
```

Comment:
NIT: instead of doing `assign(scripts: ...)` and then `assign_html()` we could merge both assigns with the already existing helper `assign_html_and_scripts()`

Reply:
Good idea